src/buildList.js

```diff
@@ -6,10 +6,11 @@ const goerli = require("./tokens/goerli.json");
 const kovan = require("./tokens/kovan.json");
 const polygon = require("./tokens/polygon.json");
 const mumbai = require("./tokens/mumbai.json");
+const bridgeUtils = require('@uniswap/token-list-bridge-utils');
 
-module.exports = function buildList() {
+module.exports = async function buildList() {
   const parsed = version.split(".");
-  return {
+  const l1List = {
```

Comment:
I don't think this needs to be an async function. Returning the `chainify` result (I'm assuming it's a promise?) will give you the same behavior.